index-pl.html

```diff
@@ -593,21 +593,21 @@ <h2>The Human Cost</h2>
       </label>
 
       <label class="row half bigsplash">
-        <div>Mask Effectiveness on Exhale: <span class="dotted">{{convertToPercent(eout)}}</span>%</div>
+        <div>Skuteczność maski przy Wydychaniu: <span class="dotted">{{convertToPercent(eout)}}</span>%</div>
```

Comment:
Nie rozumiem czemu Wydychaniu jest z wielkiej litery. Help.

Reply:
Też nie wiem, skopiowałem z oryginału. Ale nie ma to większego sensu, więc zmieńmy na małą.